src/models/utils/displayableFilter.ts

```diff
@@ -0,0 +1,17 @@
+import { Filter } from '@yext/answers-core';
+
+/**
+ * A {@link Filter} with additional fields meant to be displayed to the end user.
+ *
+ * @public
+ */
+export interface DisplayableFilter extends Filter {
+  /**
+   * Whether or not the filter is selected.
+   */
+  selected?: boolean,
+  /**
+   * The filter's display name.
+   */
+  displayName?: string
```

Comment:
should displayName and selected be optional?

Reply:
based on previous discussions where headless may have use cases without UI involved, I thought making displayName optional might be more flexible. But yeah selected should be required